src/wallet/components/WalletDropdownDisconnect.tsx

```diff
@@ -8,13 +8,10 @@ export function WalletDropdownDisconnect({
   className,
   text = 'Disconnect',
 }: WalletDropdownDisconnectReact) {
-  const { connectors } = useConnect();
-  const { disconnect } = useDisconnect();
-  const connector = connectors[0];
-
+  const { disconnect, connectors } = useDisconnect();
   const handleDisconnect = useCallback(() => {
-    disconnect({ connector });
-  }, [connector, disconnect]);
+    connectors.map((connector) => disconnect({ connector }));
```

Comment:
Oh nice, can we add a line a comment that explain the why we do this.
It will help future developers in mantaining this.